src/utils/crypto.js

```diff
@@ -140,3 +143,49 @@ export function tweakSigner(signer) {
         network: bitcoin.networks.bitcoin,
     });
 }
+
+export const parseOutpoint = (outpoint) => {
+    const rawVout = outpoint.slice(-8);
+    const txid = outpoint
+        .substring(0, outpoint.length - 8)
+        .match(/[a-fA-F0-9]{2}/g)
+        .reverse()
+        .join("");
+
+    const buf = new ArrayBuffer(4);
+    const view = new DataView(buf);
+    rawVout.match(/../g).forEach((b, i) => {
+        view.setUint8(i, parseInt(b, 16));
+    });
+
+    const vout = view.getInt32(0, 1);
+    return [txid, vout];
+};
+
+export function sortUtxos(utxos) {
+    const sortedData = utxos.sort((a, b) => b.status.block_time - a.status.block_time);
+    return sortedData.map((utxo) => ({ ...utxo, key: `${utxo.txid}:${utxo.vout}` }));
+}
+
+export const getInscriptionsForAddress = async (address) => {
+    const response = await axios.get(`${TURBO_API}/wallet/${address}/inscriptions`);
+    return response.data;
+};
+
+export const getOutpointFromCache = async (inscriptionId) => {
+    const key = `${LocalStorageKeys.INSCRIPTIONS_OUTPOINT}:${inscriptionId}`;
```

Comment:
nit: better in inscriptions.js?

Reply:
think it keeps things cleaner, and quite like this segregation of responsibilities within files

Reply:
agree